doc/src/features/LinkZeitraumPraktikumsstellen.md

```diff
@@ -0,0 +1,8 @@
+## Link between [Praktikumsstellen](../glossary.md#praktikumsstelle) and [Meldezeitraum](../glossary.md#meldezeitraum)
```

Comment:
i am not sure if it is really a new feature or an an update for `[Report Praktikumsstelle](MeldungOertlAusbilder.md)`